### DISCUSSION TITLE
Submitting academic work to https://qualcoder.wordpress.com/

### DISCUSSION THREAD

**Discussion by ChefDani** [ccbogel/QualCoder#974] [General]
At the website of Qualcoder, there is a subheading revolving 'studies using qualcoder'.
I just succesfully defended my thesis in which I used Qualcoder 3.5 to effectively code the available data.

How do I reach somebody to submit my work to this wordpress website?

**Comment by ccbogel** [comment]
Easy - tell me about it. Cite your work here or via email and I will update the Wordpress site. 
I hope QualCoder worked well enough for your needs.
regards Colin

**Reply by ChefDani** [reply]
Dear Colin,

Attached my thesis in this comment.

In fortitude,
 Dani Smith

[Uploading Thesis 1.10 UT.pdf…]()

**Reply by ChefDani** [reply]
PS hereby my thesis properly inserted 

[Thesis 1.10 UT.pdf](https://github.com/user-attachments/files/18614723/Thesis.1.10.UT.pdf)

**Comment by ccbogel** [comment]
Thank you very much.

with regards Colin
________________________________

**Reply by ChefDani** [reply]
For completeness hereby also the direct link of the University of Twente repository: https://essay.utwente.nl/104803/

**Comment by ccbogel** [comment]
Thank you Dani,

I quickly read through your thesis.
I hope you found QualCoder useful and can promote it to your colleagues.

with regards Colin Curtain
________________________________
